sdcm/send_email.py

```diff
@@ -122,16 +152,52 @@ def send_email(self, subject, content, html=True, files=()):
                              self.send_email, self.email_recipients)
 
     def send_report(self, results):
-        try:
-            email_data = self.build_data_for_render(results)
-            self.log.info('Send email with results to {}'.format(self.email_recipients))
-            html, attached_files = self.build_report(email_data)
-            self.send_email(subject=email_data['subject'], content=html, files=attached_files)
-        except Exception as details:  # pylint: disable=broad-except
-            self.log.error("Error during sending email: %s", details, exc_info=True)
+        report_data = self.build_data_for_report(results)
+        attachments_data = self.build_data_for_attachments(results)
+        for _ in range(10):
```

Comment:
why do you need to iterate 10 times?
(if you need retry mechanism use retry decorator)

Reply:
anyway, I don't see a need for it. make it simple, try once